app/models/obituary.rb

```diff
@@ -3,8 +3,25 @@ class Obituary < ApplicationRecord
   belongs_to :user
   before_save :default_image
 
+  def name
+    first_name + ' ' + last_name
+  end
+
+  def location
+    city + ' ' + state
+  end
+
+  def date
+    created_at.strftime("%B %-d, %Y")
+  end
+
   private
   def default_image
-    self.image_url ||= '/thumbnail_logo.png'
+    self.image_url ||= 'https://image.shutterstock.com/image-vector/ui-image-placeholder-wireframes-apps-260nw-1037719204.jpg'
```

Comment:
Not only did thumbnail_logo not work, but when I create a new obituary on local host an image doesn't display at all. I saw that it worked on Heroku during the demo today to use the "rescue image" so I'm not really sure what is going on.